manifests/r/RescueTime,Inc/RescueTime/2.16.8.2/RescueTime,Inc.RescueTime.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RescueTime,Inc.RescueTime
+PackageVersion: 2.16.8.2
+PackageLocale: en-US
+Publisher: RescueTime, Inc.

```

This must match what is shown in control panel
```suggestion
Publisher: RescueTime.com
```